avocado/core/test.py

```diff
@@ -383,6 +384,11 @@ def _start_logging(self):
                                                        formatter,
                                                        self._ssh_logfile)
 
+        if isinstance(sys.stdout, output.LoggingFile):
+            sys.stdout.add_logger(logging.getLogger("avocado.test.stdout"))
+        if isinstance(sys.stderr, output.LoggingFile):
+            sys.stderr.add_logger(logging.getLogger("avocado.test.stderr"))
+
```

Comment:
OK, when I replaced the first commit with the detection and added the second commit, it still failed not using the first commit workaround, therefor this does not fixes the logging issue https://travis-ci.org/avocado-framework/avocado/jobs/178818855#L4448